web-marketplace/src/index.tsx

```diff
@@ -117,33 +118,34 @@ root.render(
         <LocalizationProvider dateAdapter={AdapterDateFns}>
           <AnalyticsProvider instance={analytics}>
             <AuthProvider>
-              <ChainProvider
-                chains={chains.filter(chain => chain.chain_name === 'regen')}
-                assetLists={assets.filter(
-                  chain => chain.chain_name === 'regen',
-                )}
-                wallets={wallets}
-                walletConnectOptions={{
-                  signClient: {
-                    projectId: import.meta.env.VITE_WALLET_CONNECT_PROJECT_ID,
-                    relayUrl: WALLET_CONNECT_RELAY_URL,
-                    metadata: walletConnectClientMeta,
-                  },
-                }}
-              >
-                <WalletProvider>
-                  <LedgerProvider>
-                    <ThemeProvider>
+              <ThemeProvider>
```

Comment:
we needed to move the ThemeProvider up so the `LoginModalMobile` provided as `walletModal` to `ChainProvider` renders with our MUI theme